tasks1.js

```diff
@@ -0,0 +1,119 @@
+// 1
+function convertMinutesToHours(minutes) {
+  return (Math.floor(minutes / 60)) + ':' + (minutes % 60);
+}
+console.log(convertMinutesToHours(75));
+
+function convertMinutesToHoursInNumber(minutes) {
+  return (Math.floor(minutes / 60)) + (minutes % 60) / 100;
+}
+console.log(convertMinutesToHoursInNumber(75));
```

Comment:
This is incorrect. Take a look at the piece of code provided in the homework:
```
console.log(hours); // 1.25
```

The result should be `1.25` which is not the case in your code.